### PR TITLE
Remove copy card IDs

### PR DESCRIPTION
**Pull request by TheLastProject**
#1585 showed that the "Copy IDs" can be confused for a "copy-paste" action.

I doubt this feature is very commonly used, I haven't used it myself even once in years.

Maybe it's better to remove it? Users can still copy the IDs per card by pressing the ID and then selecting it. Sure, they can't as easily copy the ID of a big amount of cards but when will you ever need to do that?

### CODE REVIEW

File: app/src/main/java/protect/card_locker/MainActivity.java [hunk 1]
```diff
@@ -90,35 +90,7 @@ public boolean onPrepareActionMode(ActionMode inputMode, Menu inputMenu) {
 
         @Override
         public boolean onActionItemClicked(ActionMode inputMode, MenuItem inputItem) {
-            if (inputItem.getItemId() == R.id.action_copy_to_clipboard) {
-                ClipboardManager clipboard = (ClipboardManager) getSystemService(CLIPBOARD_SERVICE);
```
Comment (obfusk) [inline app/src/main/java/protect/card_locker/MainActivity.java:94]: This leaves a few unused imports like `android.content.ClipData` and `android.content.ClipboardManager` (haven't checked for more).

**Comment by obfusk** [comment]
I didn't test it or look at all the modified translations, but the changes LGTM (apart from maybe removing the unused imports).

**Review by obfusk — APPROVED** [review]

**Comment by TheLastProject** [comment]
I assume nobody will miss this, but we'll see. If someone misses it and can explain a good use case to bring it back I can always reconsider this.

**Comment by obfusk** [comment]
> I assume nobody will miss this, but we'll see. If someone misses it and can explain a good use case to bring it back I can always reconsider this.

Yeah, that makes sense to me. A concrete use case would hopefully also allow us to find the best way to provide the feature, making it easy to find for someone with that use case, without confusing others. And allow us to decide on the best way to handle copying multiple IDs.

**Pull request merged**